From a7724b21f7e29151e53540d3f7c7f5add6be0fdb Mon Sep 17 00:00:00 2001
Message-Id: <a7724b21f7e29151e53540d3f7c7f5add6be0fdb.1659883483.git.plcl@users.sourceforge.net>
From: =?UTF-8?q?Pedro=20L=C3=B3pez-Cabanillas?= <plcl@users.sourceforge.net>
Date: Sun, 7 Aug 2022 16:43:50 +0200
Subject: [PATCH] disable tests

---
 CMakeLists.txt | 36 ------------------------------------
 1 file changed, 36 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eff6d48..1266c70 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,21 +29,6 @@ option(BUILD_SONIVOX_SHARED "Build the shared library" TRUE)
 
 include(GNUInstallDirs)
 
-find_package(GTest CONFIG)
-if (NOT GTest_FOUND)
-    message( STATUS "GTest not found. Fetching the git repository..." )
-    set( INSTALL_GTEST OFF CACHE BOOL "Enable installation of gooogletest" FORCE )
-    set( BUILD_GMOCK OFF CACHE BOOL "Builds the googlemock subproject" FORCE )
-    include( FetchContent )
-    FetchContent_Declare( googletest
-      GIT_REPOSITORY https://github.com/google/googletest.git
-      GIT_TAG release-1.12.1
-    )
-    # For Windows: Prevent overriding the parent project's compiler/linker settings
-    set( gtest_force_shared_crt ON CACHE BOOL "" FORCE )
-    FetchContent_MakeAvailable( googletest )
-endif()
-
 if (UNIX AND NOT APPLE)
     find_library(MATH_LIBRARY m)
     message(STATUS "Math library: ${MATH_LIBRARY}")
@@ -229,24 +214,3 @@ install( FILES
     "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake"
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
 )
-
-# Unit testing
-
-enable_testing()
-
-add_executable( SonivoxTest
-  test/SonivoxTest.cpp
-)
-
-target_include_directories( SonivoxTest PRIVATE
-    arm-wt-22k/include
-    fakes
-)
-
-target_link_libraries( SonivoxTest
-  GTest::gtest_main
-  sonivox
-)
-
-include( GoogleTest )
-gtest_discover_tests( SonivoxTest EXTRA_ARGS "-P${PROJECT_SOURCE_DIR}/test/res/" DISCOVERY_TIMEOUT 300 ) 
-- 
2.16.4

