--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,13 +49,12 @@ else()
     find_package(QT NAMES Qt5 Qt6)
 endif()
 
-if(QT_VERSION VERSION_LESS 6.0.0)
-    find_package (Qt5 COMPONENTS Gui Widgets PrintSupport LinguistTools REQUIRED)
-else()
-    find_package (Qt6 COMPONENTS Gui Widgets PrintSupport LinguistTools Core5Compat REQUIRED)
+find_package (Qt${QT_VERSION_MAJOR} COMPONENTS Gui Widgets PrintSupport LinguistTools REQUIRED)
+if(QT_VERSION VERSION_GREATER_EQUAL 6.0.0)
+    find_package (Qt6Core5Compat REQUIRED)
 endif()
 
-find_package (Drumstick 2.5 COMPONENTS File RT Widgets REQUIRED)
+find_package (Drumstick 2.6 COMPONENTS File RT Widgets REQUIRED)
 
 find_package ( PkgConfig )
 pkg_check_modules( UCHARDET REQUIRED IMPORTED_TARGET uchardet )
